controller/backing_image_data_source_controller.go

```diff
@@ -669,7 +682,11 @@ func (c *BackingImageDataSourceController) generateBackingImageDataSourcePodMani
 		"--source-type", string(bids.Spec.SourceType),
 	}
 
-	if err := c.prepareRunningParameters(bids); err != nil {
+	bids.Status.RunningParameters = bids.Spec.Parameters
```

Comment:
I think it is because besides the parameters provided by the users 
we have some parameters derived after
this `RunningParameters` is originally for `export-from-volume`
the parameter users provided for `export-from-volume` is `Volume Name`
But we derived others parameters for the execution
```
	DataSourceTypeExportFromVolumeParameterVolumeName                = "volume-name"    // from users directly

// derived after
	DataSourceTypeExportFromVolumeParameterVolumeSize                = "volume-size"
	DataSourceTypeExportFromVolumeParameterSnapshotName              = "snapshot-name"
	DataSourceTypeExportFromVolumeParameterSenderAddress             = "sender-address"
	DataSourceTypeExportFromVolumeParameterFileSyncHTTPClientTimeout = "file-sync-http-client-timeout"
```
You can see the function `prepareRunningParametersForExport()`


Same reason, we get the backing image UUID for running parameters inside the function
because it is inconvenient for users to fill in the UUID when cloning